src/atoms/Text/Text.stories.js

```diff
@@ -19,4 +19,25 @@ storiesOf('Text', module)
     <Text size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
     </Text>
+  ))
+  .add('as span', () => (
+    <Text size={select('Size', sizeOptions, 'large')} as="span">
+      {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
+    </Text>
+  ))
+  .add('show content in only 1 line and hide the rest ', () => (
+    <Text size={select('Size', sizeOptions, 'large')} truncate maxLines={1} _lineHeight="medium">
```

Comment:
all ```_``` props like ```_lineHeight``` are private props are we should not use it unless it's an exception from the design end. Can we not use it here?

Reply:
like rename the props ? _truncate or _maxLines ?

Reply:
No, I'm saying don't use _lineHeight. Line height is not something we should manually define here. It is only exposed so that we can use it in internal components like Button where we have no choice but to change the line-height.

Reply:
done